src/context.ts

```diff
@@ -335,6 +336,35 @@ export class Context<
     return this.newResponse(null, status)
   }
 
+  streamText = (
+    cb: (stream: StreamingApi) => Promise<void>,
+    arg?: StatusCode | ResponseInit,
+    headers?: HeaderRecord
+  ): Response => {
+    headers ??= {}
+    headers['content-type'] = 'text/plain; charset=UTF-8'
+    headers['x-content-type-options'] = 'nosniff'
+
+    return this.stream(cb, arg, headers)
+  }
+
+  stream = (
+    cb: (stream: StreamingApi) => Promise<void>,
+    arg?: StatusCode | ResponseInit,
+    headers?: HeaderRecord
+  ): Response => {
+    const { readable, writable } = new TransformStream()
+    const stream = new StreamingApi(writable)
+    cb(stream).finally(() => stream.close())
+
+    this._pH ??= {}
+    this._pH['transfer-encoding'] = 'chunked'
```

Comment:
How about specifying `transfer-encoding: chunked` in `c.streamText()` without setting the header in `c.steam()`? This will make `c.stream()` more generic.

Reply:
I agree. Fixed!